bindings/wasm/.eslintrc.js

```diff
@@ -20,5 +20,6 @@ module.exports = {
             'error',
             { 'ts-ignore': 'allow-with-description' },
         ],
+        '@typescript-eslint/no-explicit-any': 'off',
```

Comment:
Pour quoi?

Reply:
Was required at some point when utils were async and I had to write something manually for wasm, but actually not needed anymore, will remove :+1: